src/Diagnostics.ModelsAndUtils/Attributes/ResourceFilter/Enumerations.cs

```diff
@@ -4,6 +4,60 @@
 
 namespace Diagnostics.ModelsAndUtils.Attributes
 {
+    public class ArmResourceType
```

Comment:
One Suggestion -These fields can be part of IResource Interface and we can define the values in Inherited Resource Objects. 
In the Controller Base Class, you have access to TResource : IResource. So you'll be able to read these values.

Then, when adding a new Resource, we dont have to define this ArmResourceType.

Reply:
Good suggestion, let me try that.